### PR TITLE
Add the model name as cpu_model to system profile

### PR DESCRIPTION
**Pull request by CodeHeeler**
Add the cpu model name as cpu_model to system profile as requested:
https://issues.redhat.com/browse/HBISPSC-39

Related puptoo PR:
https://github.com/RedHatInsights/insights-puptoo/pull/117

### CODE REVIEW

File: schemas/system_profile/v1.yaml [hunk 1]
```diff
@@ -139,6 +139,11 @@ $defs:
         pattern: '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}'
         maxLength: 36
         example: 22cd8e39-13bb-4d02-8316-84b850dc5136
+      cpu_model:
+        description: The cpu model name
+        type: string
+        max_Length: 100
```
Comment (iphands) [inline schemas/system_profile/v1.yaml:145]: should be max_length the L is capitalized here

**Review by iphands — CHANGES_REQUESTED** [review]

**Review by ryandillinfelton — APPROVED** [review]

**Review by kruai — APPROVED** [review]

**Review by tahmidefaz — CHANGES_REQUESTED** [review]
@CodeHeeler Sorry, I just got to look at this. Can you provide two very simple examples of valid and invalid `cpu_model` [here](https://github.com/RedHatInsights/inventory-schemas/tree/master/tests/utils)? It will help us with not writing tests in HBI for this specific field.

**Comment by CodeHeeler** [comment]
@tahmidefaz Ah, thanks!  Done. Will do this in the future for all new facts.

**Comment by kruai** [comment]
I've dismissed old reviews, as Dana has addressed all the requested changes.

**Review by tahmidefaz — APPROVED** [review]
Awesome! Looks great!! :+1:

**Review by ryandillinfelton — APPROVED** [review]
:+1:

**Pull request merged**